packages/design-system/src/components/HelpDrawer/HelpDrawer.jsx

```diff
@@ -88,9 +129,32 @@ HelpDrawer.defaultProps = {
   closeButtonText: 'Close',
   headingLevel: '3',
 };
+
+/**
+ * Defines the shape of an analytics event for tracking that is an object with key-value pairs
+ */
+const AnalyticsEventShape = PropTypes.shape({
```

Comment:
this makes sense here for now, but eventually this should be exported and shared across different components using Analytics, and ideally this would be typed with Typescript

Reply:
Oh nice, I see you already have a Typescript definition for this already defined in Analytics.ts